app/layout.tsx

```diff
@@ -32,7 +32,7 @@ const RootLayout = ({
 }): JSX.Element => {
   return (
     <html lang="en" className={GeistSans.className}>
-      <body className="dark:dark bg-background px-4 pb-8 text-foreground">
+      <body className="dark:dark bg-background px-4 text-foreground">
```

Comment:
I did not think of this before; but what if we added `pb-8` on breakpoints smaller than `xl` and then at `xl` removed the `pb-8`?

```suggestion
      <body className="dark:dark bg-background px-4 pb-8 text-foreground xl:pb-0 ">
```